modules/con-sdpolicy-account-management.adoc

```diff
@@ -6,48 +6,37 @@
 = Account management
 
 == Billing
-Each {product-title} cluster requires a minimum annual base cluster purchase and there are two billing options available for each cluster, Standard and Customer Cloud Subscription (CCS).
+{product-title} is billed through Amazon Web Services (AWS) based on usage of AWS components used by the service, such as load balancers, storage, EC2 instances, and so on.
 
-Standard {product-title} clusters are deployed in to their own cloud infrastructure accounts, each owned by Red Hat. Red Hat is responsible for this account, and cloud infrastructure costs are paid directly by Red Hat. The customer will only pay the Red Hat subscription costs.
-
-In the CCS model, the customer pays the cloud infrastructure provider directly for cloud costs and the cloud infrastructure account will be part of a customer’s Organization, with specific access granted to Red Hat. The customer will have restricted access to this account, but will be able to view billing and usage information. In this model, the customer will pay Red Hat for the CCS subscription and will pay the cloud provider for the cloud costs. It is the customer's responsibility to pre-purchase or provide Reserved Instance (RI) compute instances to ensure lower cloud infrastructure costs.
-
-Additional resources may be purchased for an OpenShift Dedicated Cluster, including:
-
-- Additional Nodes (must be same type/size as existing application nodes)
-- Middleware (JBoss EAP, JBoss Fuse, etc.) - additional pricing based on specific middleware component
-- Additional Storage in increments of 500GB (standard only)
-- Additional 12 TiB Network I/O (standard only)
-- Load Balancers for Services are available in bundles of 4; enables non-HTTP/SNI traffic or non-standard ports (standard only)
+Any additional Red Hat software must be purchased separately.
 
 == Cluster self-service
 
-Customers can create, scale, and delete their clusters from link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)], provided they've pre-purchased the necessary subscriptions.
-
-== Cloud providers
-
-{product-title} offers OpenShift Container Platform clusters as a managed service on the following cloud providers:
-
-- Amazon Web Services (AWS)
-- Google Cloud Platform (GCP)
+Customers can create, scale, and delete their clusters either from link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] or the `rosa` CLI utility.
```

Comment:
If you're using the standard attributes file, I suggest using {cloud-redhat-com} to render "Red Hat OpenShift Cluster Manager"

Reply:
I believe this is correct as is, based on the output I've gotten from Eng recently.